app/code/core/Mage/Core/Model/App/Emulation.php

```diff
@@ -88,9 +90,9 @@ public function startEnvironmentEmulation($storeId, $area = Mage_Core_Model_App_
         $initialLocaleCode = $this->_emulateLocale($storeId, $area);
 
         $initialEnvironmentInfo = new Varien_Object();
-        $initialEnvironmentInfo->setInitialTranslateInline($initialTranslateInline)
-            ->setInitialDesign($initialDesign)
-            ->setInitialLocaleCode($initialLocaleCode);
+        $initialEnvironmentInfo->setData('initial_translate_in_line', $initialTranslateInline)
```

Comment:
As far as I read this line it should be

`$initialEnvironmentInfo->setData('initial_translate_inline', $initialTranslateInline)`

Reply:
Will be reverted.